### DISCUSSION TITLE
assets 404 after following install guide for importmaps

### DISCUSSION THREAD

**Discussion by itsyoshio** [ruby-ui/ruby_ui#141] [Q&A]
After following the [Installation Guide](https://phlexui.com/docs/installation/rails_importmaps) for the Importmaps variant, any page load results in loads of index.js and other 404 errors, all of various *.js files.

These seemingly start with Calendar Components but we're talking about various components (~300 of them).

![image](https://github.com/PhlexUI/phlex_ui/assets/12760988/b43d8fde-6f23-4f3a-ab0b-826e8335b45c)

**Comment by cirdes** [comment]
@itsyoshio, I recommend getting the stimulus controllers from https://github.com/PhlexUI/phlex_ui_stimulus/tree/main/controllers and copying them to your project. That's how I'm using them.

**Comment by itsyoshio** [comment]
In what way does this alleviate the Importmaps issues with date-fns?

Are you suggesting to not `import 'phlex_ui';` into `app/javascript/application.js` ?

**Comment by sethhorsley** [comment]
We are moving things around with the new version and I have not had time to test out as much as i want with importmaps.
I will be doing that in the next couple days. Let me know if you were able to figure it out.
